CODEOWNERS

```diff
@@ -23,6 +23,7 @@ CHANGELOG.md @AzureAD/AppleIdentityTeam @AzureAD/MSAL-ObjC-CIAM
 /MSAL/MSAL.xcodeproj/xcshareddata/xcschemes/MSAL\ Mac\ Native\ Auth\ E2E\ Tests.xcscheme @AzureAD/MSAL-ObjC-CIAM
 /Package.swift @AzureAD/MSAL-ObjC-CIAM
 /spm-integration-test.sh @AzureAD/MSAL-ObjC-CIAM
+/azure_pipelines/ @AzureAD/MSAL-ObjC-CIAM
```

Comment:
Can you add the @AzureAD/AppleIdentityTeam team too as owner?
And please move this line at line 8 (under changelog.md). 
Can you also update the comment at line 5?